AirCasting/CreateSessionViews/CreateSessionDetails/CreateSessionDetailsViewModel.swift

```diff
@@ -64,8 +68,12 @@ class CreateSessionDetailsViewModel: ObservableObject {
     
     func compareIsIndoor(sessionContext: CreateSessionContext) -> CreateSessionContext {
         sessionContext.isIndoor = isIndoor
-        isLocationSessionDetailsActive = !isIndoor
-        isConfirmCreatingSessionActive = isIndoor
+        guard locationHandler.isLocationDenied() && !(sessionContext.isIndoor != nil) else {
```

Comment:
`!(sessionContext.isIndoor != nil)` what do we want to check here? I think `sessionContext.isIndoor == nil` would be easier to read.  However, do we want to move forward to `isLocationScreenNedeed = true` if location isn't denied but isIndoor is nil?

Reply:
We were checking that to be sure that for the first session context must not be nil before checking the whole guard statement. After that, we were ensuring ourselves about location and indoor. I changed it anyway 😄. Thanks for asking